packages/hawtio/src/plugins/camel/icons/components/CamelImageIcon.tsx

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
I don't see value in putting this component under `components/`.

Generally, in this project I try to avoid structuring packages based on functionality/kind but rather based on feature/domain:
https://medium.com/sahibinden-technology/package-by-layer-vs-package-by-feature-7e89cde2ae3a

So I try to avoid creating directories such as `components`, `services`, `types`, etc under each plugin. Rather we can organise files with directories like `featureA`, `featureB`, ... I think `icons` or `images` are fine as they stand on its own.